src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -166,22 +174,22 @@ export const CustomerQuotaScreen: FunctionComponent<CustomerQuotaProps> = ({
   ]);
```

Comment:
Perhaps we can add another breadcrumb here to track the `quotaState` like how it's done with `cartState`? Not sure if it will add value, what do you think?